lib/pheme/queue_poller.rb

```diff
@@ -22,6 +30,20 @@ def initialize(queue_url:, connection_pool_block: false, max_messages: nil, form
         skip_delete: true, # manually delete messages
       }.merge(poller_configuration || {})
 
+      @poller_configuration[:idle_timeout] = idle_timeout unless idle_timeout.nil?
+
+      if message_handler
+         if message_handler.ancestors.include?(Pheme::MessageHandler)
+            @message_handler = message_handler
+         else
+           raise ArgumentError, 'Invalid message handler, must inherit from Pheme::MessageHandler'
+         end
+      end
+
+      @block_message_handler = block_message_handler if block_message_handler
```

Comment:
```suggestion
      @block_message_handler = block_message_handler
```

This `if` condition won't affect the result of this assignment.